src/components/Form/Form.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Form.scss';
+import 'bulma';
+
+export class Form extends React.Component {
+  state = {
+    users: [...this.props.users],
+    id: 3,
+    selectedUserId: 0,
+    inputTitle: '',
+    inputError: false,
+    selectError: false,
+  };
+
+  inputTitleChange = (event) => {
+    const { target: { value } } = event;
+
+    this.setState({
+      inputTitle: value,
+      inputError: false,
+    });
+  };
+
+  selectChange = (event) => {
+    const { target: { value } } = event;
+
+    this.setState({
+      selectedUserId: +value,
+      selectError: false,
+    });
+  };
+
+  buttonSubmit = (event) => {
```

Comment:
one setState for one method, write all that code below in one setState

Reply:
could you give me a link to an article, how should I did this